packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -91,18 +76,23 @@ export class TSClient implements Generatable {
       generator.config.engineType = engineType
     }
 
+    const inlineSchema = (await readFile(schemaPath)).toString('base64')
+    const inlineSchemaHash = crypto.createHash('sha256').update(inlineSchema).digest('hex')
```

Comment:
Inline schema and hash needed for the data proxy to work correctly.

Reply:
What's the impact of this on `prisma generate` for big schemas? since we would do this all the time even when not actually needed? (user using Node.js only, without accelerate)

I'm thinking that big schemas that were slow to generate would get worse? 🤔

Reply:
Turns out there is no impact.